src/librustc/ty/mod.rs

```diff
@@ -2380,6 +2380,17 @@ impl<'a, 'gcx, 'tcx> TyCtxt<'a, 'gcx, 'tcx> {
     }
 }
 
+impl<'a, 'tcx> TyCtxt<'a, 'tcx, 'tcx> {
+    /// Given a DefId and some Substs, produces the monomorphic item type.
```

Comment:
This shouldn't exist - maybe inline the body or use `Instance::new(def_id, substs).ty(tcx)` instead?

Reply:
Oh sorry I should have looked closer, I am removing it now.